contracts/dapp/Application.sol

```diff
@@ -41,6 +41,10 @@ contract Application is
     /// @dev See the `getConsensus` and `migrateToConsensus` functions.
     IConsensus internal _consensus;
 
+    /// @notice The data availability solution.
+    /// @dev See the `getDataAvailability` function.
+    bytes internal _dataAvailability;
```

Comment:
why bytes?

Reply:
I mean, why not a ERC-165 contract?

Reply:
Because with `bytes` we can encode more data specific to that application.
For example, applications that use Espresso would specify their namespace and Espresso block height.